### DISCUSSION TITLE
Sandık çilesi

### DISCUSSION THREAD

**Discussion by Bulent3407** [IBAX-io/go-ibax#1657] [Q&A] [answered]
Sandıktan en yüksek kaç çıktı?Bana hep 100 altı geldi

**Comment by Eddyhunter07** [comment, marked as answer]
260 en fazla

**Reply by Bulent3407** [reply]
İyimiş 260 bana gelse uçurur

**Comment by THECROVV** [comment]
200 en yuksek sanırım ya da 157, en düşük ise 51 çıktı. 147 de çıkmış 63 de çıkmış ama  ortalama  100 civarı diyebiliriz.

**Reply by Bulent3407** [reply]
👍👍
